lib/SPIRV/SPIRVRegularizeLLVM.cpp

```diff
@@ -400,13 +375,44 @@ bool SPIRVRegularizeLLVMBase::regularize() {
         // a scalar/vector integers, so we have to extend such arguments
         if (II.isLogicalShift() &&
             II.getOperand(0)->getType()->isIntOrIntVectorTy(1)) {
-          auto *NewInst = extendBitInstBoolArg(&II);
-          for (auto *U : II.users()) {
-            if (cast<Instruction>(U)->getOpcode() == Instruction::ZExt) {
-              U->dropAllReferences();
-              U->replaceAllUsesWith(NewInst);
-              ToErase.push_back(cast<Instruction>(U));
+          IRBuilder<> Builder(&II);
```

Comment:
Can you please add example LLVM code as comment? Input IR followed by IR generated by your transformation?
I understand the testcase covers it. But this comment can help in documentation. Thanks